tests/e2e/specs/modules/search-console/dashboard-date-range.test.js

```diff
@@ -52,7 +62,18 @@ describe( 'date range filtering on dashboard views', () => {
 
 		await page.setRequestInterception( true );
 		useRequestInterception( ( request ) => {
-			if ( request.url().match( 'google-site-kit/v1/data/' ) ) {
+			const url = request.url();
+			// Widget API requests. As mentioned above, these can be
+			// refactored to use the mock response utility as part of
+			// https://github.com/google/site-kit-wp/issues/2586.
```

Comment:
These comments are nice, but we should not rely on these alone to remember to do this. It would be great to leave comments on such referenced issues with links to these comments to increase our awareness of these changes to be made.